src/client/debugger/jupyter/helper.ts

```diff
@@ -5,17 +5,24 @@ import { DebugProtocol } from 'vscode-debugprotocol';
 import { IKernel } from '../../datascience/jupyter/kernels/types';
 import { IKernelDebugAdapterConfig, KernelDebugMode } from '../types';
 
-export async function isUsingIpykernel6OrLater(kernel: IKernel): Promise<boolean> {
+export enum IpykernelCheckResult {
+    Unknown,
+    Ok,
+    Missing,
+    NoKernel
```

Comment:
Actually, NoKernel shouldn't be in this enum if it isn't returned by the function below. This made more sense earlier when this was all done in one file, then I merged it into some other changes that moved stuff around. I'm going to change this a bit